src/radical/pilot/agent/bootstrap_0.sh

```diff
@@ -176,7 +176,7 @@ profile_event()
 {
     PROFILE="bootstrap_0.prof"
 
-    if test -z "$RADICAL_PILOT_PROFILE"
+    if test -z "$RADICAL_PILOT_PROFILE$RADICAL_PROFILE"
```

Comment:
Forward compatibility to the backward compatibility of the upcoming release ;-)

Reply:
This test if any of the two variables exist, right?

Reply:
yes.